contracts/DToken.sol

```diff
@@ -70,7 +69,8 @@ contract DToken is IDToken, Context {
   function transfer(address recipient, uint256 amount) public virtual override returns (bool) {
     recipient;
     amount;
-    revert TokenErrors.DTokenTransferNotAllowed();
+    require(true != true, 'DTokenTransferNotAllowed');
+    revert();
```

Comment:
I think, `revert()` is never called.

Reply:
Although these ERC20 functions are not used, it was necessary to specify the return type to override the interface of ERC20. However, the solidity compiler indicates an error for these expressions. These revert state were for removing errors in the Solidity compiler. Instead, I wrote meaningless return values!